collective/js/fancytree/profiles/theme-lion/cssregistry.xml

```diff
@@ -11,6 +11,6 @@
              rendering="import"
              expression=""
              applyPrefix="1"
-             id="++resource++collective.js.fancytree/ui.fancytree.css"/>
```

Comment:
Hi @tdesvenain 

to me it still misses an upgrade step, all this can be done using GS xml files :
- add a upgrade_to_12 GS profile;
- add a cssregistry.xml removing the old registered ressource;
- add a metadata.xml file that makes it depend on :default and :vista profiles.

Or at least again explain it on the package README.rst

Gauthier